main.rb

```diff
@@ -52,15 +68,15 @@ def option_chosen(option)
     when '8'
```

Comment:
Please add an implementation that would allow us to list labels. The list of labels should be present in a json file.